addons/crm/models/crm_team.py

```diff
@@ -228,3 +228,18 @@ def _graph_title_and_key(self):
         if self.dashboard_graph_model == 'crm.lead':
             return ['', _('Pipeline: Expected Revenue')] # no more title
         return super(Team, self)._graph_title_and_key()
+
+    @api.multi
+    def unlink(self):
+        mandatory_xml_ids = [
+            'sales_team.team_sales_department',
+            'sales_team.crm_team_1']
```

Comment:
This one is a demo data, not necessary to check it.

Reply:
I tried to remove Europe and America team, removing any of the two makes the installation of sale_management impossible. Do I still remove crm_team_1 ?

Reply:
If I understand correctly, it will fail only in case you are trying to install sale_management with demo data. In production, you should not have demo data, so that's not a big deal.